src/lib/misc.c

```diff
@@ -1153,8 +1153,12 @@ grabdate(const char *s, int64_t *t)
 
     if (!compiled) {
         compiled = 1;
-        (void) regcomp(
-          &r, "([0-9][0-9][0-9][0-9])[-/]([0-9][0-9])[-/]([0-9][0-9])", REG_EXTENDED);
+        if (regcomp(&r,
+                    "([0-9][0-9][0-9][0-9])[-/]([0-9][0-9])[-/]([0-9][0-9])",
+                    REG_EXTENDED) != 0) {
+            fprintf(stderr, "Can't compile regex\n");
```

Comment:
What I've meant is to add error code that's returned from regcomp. What you can also do is to use RNP_LOG(), so that if it happnes developer gets file name and line number (or add this info manually)